dist/utilities/flexbox/README.md

```diff
@@ -135,14 +135,23 @@ Allow a flex child to grow and shrink as needed. This means the flex child
 does so according to the following setting: `flex: 1 1 auto`.
 
 
+## `flex-initial`
+
+| Parameters | Type | Example |
+| ---------- | ---- | ------- |
+| `$important` | `boolean` | (optional) Default: false |
+
+Ensure the flex children do collapse to their contents size. This means the flex
```

Comment:
Can this language be tweak to reflect that the flex children can shrink to less than its content size?